### ISSUE TITLE
Asking for flash briefing plays same audio lots of times

### ISSUE THREAD

**Issue by AidanLovelace** [ac87/GoogleAssistantWindows#1]
When you ask google for your flash briefing, it plays the same sound over itself.

**Comment by ac87** [comment]
Good catch, I didn't try that one. Playback now only starts once the conversation response is finished so all the audio is in the buffer.
